source/Camera/Basic/CaptureTutorial.md

```diff
@@ -0,0 +1,214 @@
+```csharp
+var zdfFile = Zivid.NET.Environment.DataPath + "/MiscObjects.zdf";
+var camera = zivid.CreateFileCamera(zdfFile);
+```
+
+---
+**Note**
+
+The quality of the point cloud you get from *MiscObjects.zdf* is not representative of the Zivid One+.
+
+---
+
+## Configure
+
+As with all cameras there are settings that can be configured. These may be set manually, or you use our Capture Assistant.
+
+### Capture Assistant
+
+It can be difficult to know what settings to configure. Luckily we have the Capture Assistant. This is available in the Zivid SDK to help configure camera settings ([go to source][captureassistant-url]).
+```csharp
+var suggestSettingsParameters = new Zivid.NET.CaptureAssistant.SuggestSettingsParameters(Duration.FromMilliseconds(1200), Zivid.NET.CaptureAssistant.AmbientLightFrequency.none);
+Console.WriteLine("Running Capture Assistant with parameters: {0}", suggestSettingsParameters);
+var settingsList = Zivid.NET.CaptureAssistant.SuggestSettings(camera, suggestSettingsParameters);
+```
+
+These settings can be used in an [HDR capture](#capture-hdr), which we will discuss later.
+
+As opposed to manual configuration of settings, there are only two parameters to consider with Capture Assistant.
+
+1. **Maximum Capture Time** in number of milliseconds.
+    1. Minimum capture time is 200ms. This allows only one frame to be captured.
+    2. The algorithm will combine multiple frames if the budget allows.
+    3. The algorithm will attempt to cover as much of the dynamic range in the scene as possible.
+    4. A maximum capture time of more than 1 second will get good coverage in most scenarios.
+2. **Ambient light compensation**
+    1. May restrict capture assistant to exposure periods that are multiples of the ambient light period.
+    2. 60Hz is found in (amongst others) Japan, Americas, Taiwan, South Korea and Philippines.
+    3. 50Hz is found in most rest of the world.
+
+### Manual configuration
+
+We may choose to configure settings manually ([go to source][settings-url]). For more information about what each settings does, please see [Zivid One+ Camera Settings][kb-camera_settings-url].
+
+#### Single Frame
+
+We can configure settings for an individual frame directly to the camera.
+```csharp
+camera.UpdateSettings(s =>
+{
+    s.Iris = 10;
+    s.ExposureTime = Duration.FromMicroseconds(10000);
+    s.Brightness = 1;
+    s.Gain = 1;
+    s.Bidirectional = false;
+    s.Filters.Contrast.Enabled = true;
+    s.Filters.Contrast.Threshold = 5;
+    s.Filters.Gaussian.Enabled = true;
+    s.Filters.Gaussian.Sigma = 1.5;
+    s.Filters.Outlier.Enabled = true;
+    s.Filters.Outlier.Threshold = 5;
+    s.Filters.Reflection.Enabled = true;
+    s.Filters.Saturated.Enabled = true;
+    s.BlueBalance = 1.081;
+    s.RedBalance = 1.709;
+});
+```
+
+#### HDR Frame
+
+We may also set a list of settings to be used in an [HDR capture](#capture-hdr).
+```csharp
+var irisList = new List<ulong>() { 20, 25, 30 };
```

Comment:
Change iris values to 14, 21, 35 like in: 4406342f5eae91e649fa393a8bb1bc9f253ae598
There is already a PR to change these values for python sample. We will make on for other languages as well.